catch "error [notafunc]
show error
